configure.ac

```diff
@@ -1023,6 +1023,7 @@ AC_SUBST([GAP_DEFINES], [$GAP_DEFINES])
 
 AC_CONFIG_FILES([GNUmakefile])
 AC_CONFIG_FILES([CITATION])
+AC_CONFIG_FILES([libgap.pc])
```

Comment:
Please also add a rule to `Makefile.rules` which regenerates this file if necessary, similar to the rule for e.g. `CITATION` in line 1500. So something like this:
```
libgap.pc: $(srcdir)/libgap.pc.in config.status
	@$(SHELL) ./config.status $@
```

Reply:
Do you also mean to add `libgap.pc` as a dependency to `install-libgap` target, where
it actually gets installed?

Reply:
done in [66eb2f7](https://github.com/gap-system/gap/pull/5080/commits/66eb2f723d707333a515c8acb59db3fb89c09d1c)